sdk/tests/conformance/extensions/s3tc-and-rgtc.html

```diff
@@ -130,6 +130,29 @@
     0x00, 0xF8, 0xE0, 0x07, 0x1B, 0x5B, 0xAB, 0xFF
 ]);
 
+// BC4 - just the alpha block from BC3 above, interpreted as the red channel.
```

Comment:
Any additional comments you could leave here regarding how you constructed this data? Pointers to docs, etc.?